code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -277,7 +280,7 @@
 	if(surplus>=0)
 		dat += "[surplus] left.<br>"
 	return dat
-
+/* // these have never been fun, like, ever.
```

Comment:
Yes they have. Staff Of Change is one of those gimmick weapons. 50/50 chance you'll kill yourself with it when you Ashdrake an angry assistant.